snuba/datasets/__init__.py

```diff
@@ -114,6 +114,15 @@ def get_bulk_loader(self, source, dest_table):
     def get_query_schema(self):
         raise NotImplementedError('dataset does not support queries')
 
+    def get_prewhere_keys(self) -> List[str]:
+        """
+        Returns the keys that will be upgraded from a WHERE condition to a PREWHERE.
+
+        This is an ordered list, from highest priority to lowest priority. So, a column at index 1 will be upgraded
+        before a column at index 2. This is relevant when we have a maximum number of prewhere keys.
+        """
+        return []
```

Comment:
I could actually make this `NotImplementedError` but don't have a strong opinion. Let me know if you do.

Reply:
I think an empty collection is a reasonable default here, personally.